### PR TITLE
Kf 37 main menu

### LINKED ISSUE #20: Style the Main Page
**Is your feature request related to a problem? Please describe.**
Main Page needs styling.

**Describe the solution you'd like**
Style the main page to match the rest of the game.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Installed [Expo Fonts package](https://www.npmjs.com/package/expo-font) 
2. Setup routing for Main Menu
3. Styled Main Menu

## Purpose
Enhance UI/UX

## Approach
Used stylesheets to do all of the stylings. Used React Native Router for all of the routing. 

Closes #20, #7

### CODE REVIEW

**Review by connielion — DISMISSED** [review]
Noice. Don't see anything wrong! Is the StatusBar still needed?

**Review by BlackCubes — DISMISSED** [review]
Looks good! Glad you guys came up with another solution

File: App.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 import { NativeRouter, Route } from 'react-router-native';
 
 export default function App() {
   return (
     <NativeRouter>
       <View style={styles.container}>
         <Route component={GamePage} path="/gamepage" exact />
+        <Route component={LevelSelect} path="/levelselect" exact />
+        <Route component={SignIn} path="/profile" exact />
         <Route component={MainMenu} path="/" exact />
```
Comment (kmausolf) [inline App.js:14]: It's so strange that the route names follow the http standard even here.  I wonder if they do this to support web versions of the app?

File: components/MainMenu.js [hunk 2]
```diff
@@ -1,13 +1,106 @@
...
\ No newline at end of file
+  const [fontLoading, setFontLoading] = useState(false);
+
+  let customFonts = {
+    SchoolBell: require("../assets/fonts/Schoolbell-Regular.ttf"),
+  };
+
+  const _loadFontsAsync = async () => {
+    await Font.loadAsync(customFonts);
+    setFontLoading(true);
```
Comment (kmausolf) [inline components/MainMenu.js:15]: I think you want to set `fontLoading` to true _before_ you make the call, then set it to false after the call has finished.

Comment (krferrell) [reply components/MainMenu.js:15]: Nice catch! Will change right now

**Pull request merged**